bot.py

```diff
@@ -140,6 +140,10 @@ def load_settings(self, data):
             await self.send_text(room,
                                  f"Sorry. I don't know what to do. Execute !help to get a list of available commands.")
 
+    @staticmethod
+    def starts_with_command(body):
+        return re.match(r"^!\w.*", body)
```

Comment:
This could return boolean to be more clear and avoid testing None. Also add comment "Checks if body starts with ! and has one or more letters after it" as regexps can be difficult to read.